src/test/java/net/sf/jabref/importer/fileformat/PdfXmpImporterTest.java

```diff
@@ -0,0 +1,70 @@
+package net.sf.jabref.importer.fileformat;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+import net.sf.jabref.importer.OutputPrinterToNull;
+import net.sf.jabref.model.entry.BibtexEntry;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+
+
+public class PdfXmpImporterTest {
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    private PdfXmpImporter importer;
+
+
+    @Before
+    public void setUp() {
+        if (Globals.prefs == null) {
+            Globals.prefs = JabRefPreferences.getInstance();
+        }
+        importer = new PdfXmpImporter();
+    }
+
+    @Test
+    public void testGetFormatName() throws Exception {
+        Assert.assertEquals("XMP-annotated PDF", importer.getFormatName());
+    }
+
+    @Test
+    public void testImportEntriesEncrypted() throws Exception {
+        InputStream isEncrypt = PdfXmpImporterTest.class.getResourceAsStream("encrypted.pdf");
+        thrown.expect(IOException.class);
+        importer.importEntries(isEncrypt, new OutputPrinterToNull());
+    }
+
+    @Test
+    public void testImportEntries() throws Exception {
+        InputStream is0 = PdfXmpImporterTest.class.getResourceAsStream("annotated.pdf");
+        List<BibtexEntry> bibtexEntries = importer.importEntries(is0, new OutputPrinterToNull());
+
+        Assert.assertEquals("how to annotate a pdf", bibtexEntries.get(0).getField("abstract"));
```

Comment:
Replace the four `bibtexEntries.get(0)` calls by creating an instance `BibtexEntry entry = bibtexEntries.get(0)` and use this.